webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -2080,7 +2081,7 @@ const AdminDefinition: AdminDefinitionType = {
                 title: defineMessage({id: 'admin.sidebar.mobileSecurity', defaultMessage: 'Mobile Security'}),
                 isHidden: it.any(
                     it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.ENVIRONMENT.MOBILE_SECURITY)),
-                    it.licensedForSku(LicenseSkus.Enterprise),
+                    it.minLicenseTier(LicenseSkus.Enterprise),
```

Comment:
We actually need to do this for all the checks `licensedForSku(LicenseSkus.Enterprise)` here to your new `minLicenseTier` as any feature that is available in enterprise or professional would need to be available also if you have an Enterprise Advanced sku. @harshilsharma63 was this missed in the original PR when we introduced the new sku?

Reply:
@marianunez yes, It seems this was missed. I'll open up a new PR to update the checks. I'll use the `minLicenseTier` defined here in the new PR. Will also need to add tests for checking this.

Reply:
Should we close this PR in favor of the other one?

Reply:
@enahum we can do that and I can continue my changes in this branch and open a new PR mentioning both your and my jira ticket

Reply:
0/5 really.. I'll leave it up to you

Reply:
Also there is this PR - https://github.com/mattermost/mattermost/pull/30882 so makes sense if we close this one and I open another one with the system console fix and ensure both PRs have the right changes.

Reply:
@enahum yeah we can close this and I'll continue on this branch.